### ISSUE TITLE
Security issue with JWT, flask-jwt and itsdangerous

### ISSUE THREAD

**Issue by hairychris** [Xk0nSid/cookiecutter-flask-restful#1]
Unfortunately itsdangerous is no longer actively maintained and I believe it is vulnerable to the recent attack against JWT (because it doesn't let you whitelist the algorithm and allows the user to select it).

There is a branch here https://github.com/mattupstate/flask-jwt/pull/51 where flask-jwt has switched across to pyjwt (which is actively maintained and has patched the above security issue) instead of itsdangerous.

**Comment by Xk0nSid** [comment]
@hairychris I see the branch but it's not yet merged with the master. Also I see Travis CI build failure. It's just an import error, I guess `jwt` is not yet included in the `requirements` file. Is this gonna be fixed and merged into the master cuz the last comment and Travis build is about a month old.

**Comment by hairychris** [comment]
Yeah the maintainer just posted that he is getting married so is very busy at that moment, more details on the issue linked above down the bottom.

I decided against flask for the project I'm working on (went with aio-http for other reasons) so I've just used pyjwt for that. I'm very happy with it so far.

I'm not sure what to say, if I understand correctly this security problem is very severe and there seem to be lots of people using that library without knowing. Unless of course I am misunderstanding something.

**Comment by Xk0nSid** [comment]
@hairychris Yeah. I saw the comment. `Aio-http` seems a good option. Did you choose that cuz your app requires async i/o or is there any other reason?

**Comment by hairychris** [comment]
Sorry I didn't see this. I needed either Websockets or SSE hence choosing aio-http. Went with SSE in the end to avoid having to use another protocol outside of HTTP.
